test/unit/question_base_test.rb

```diff
@@ -6,11 +6,11 @@ class QuestionBaseTest < ActiveSupport::TestCase
   end
 
   context '#next_node' do
-    should 'ensure single next node key is supplied if next_node called without block' do
+    should 'raise exception if next_node is called without a block' do
```

Comment:
I'm not quite sure what you're getting at here. I don't mention "calling a block" in the test names.

However, I do mention "calling next_node *with* a block" and "calling next_node *without* a block". Is there something about this that doesn't make sense? If so, what?

Here's some example code which might help clarify what I mean when I say a method is called with/without a block:

```ruby
def foo
  if block_given?
    puts "foo method called *with* block"
  else
    puts "foo method called *without* block"
  end
end

foo() # => foo method called *without* block

foo do
  # some code
end
# => foo method called *with* block 
```